mteb/tasks/STS/multilingual/SemRel24STS.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskSTS, MultilingualTask, TaskMetadata
+
+_LANGUAGES = {
+    "afr": ["afr-Latn"],
+    "amh": ["amh-Ethi"],
+    "arb": ["arb-Arab"],
+    "arq": ["arq-Arab"],
+    "ary": ["ary-Arab"],
+    "eng": ["eng-Latn"],
+    "hau": ["hau-Latn"],
+    "hin": ["hin-Deva"],
+    "ind": ["ind-Latn"],
+    "kin": ["kin-Latn"],
+    "mar": ["mar-Deva"],
+    "tel": ["tel-Telu"],
+}
+
+_SPLITS = ["dev", "test"]
```

Comment:
```suggestion
_SPLITS = ["test"]
```
test seems big enough (this makes the benchmark a bit faster to run.